locales/en.yml

```
@@ -28,28 +28,39 @@ myprofile:
   posts:
     add: Write a post
   title: My Profile
-chat:
+  chat:
   title: Chat
-  welcome: Start conversation with something nice.
-  with: Chat with {username}
-  dateDiff: "{diff} ago"
+  conversation: 
+      new: Start conversations and they will appear here.
+     welcome: Start conversation with something nice.
+  message:
+      new: Write a message
+      writer: by you
+      submit: Send
+      unread:  haven't opened this chat yet
+      receiver: Chat with {receiver.username}
+ lastsaw: opened chat last time
```

If there are already variables with some name, just re-use them. Also please use camelCase. Also there is again indentation problem.

```suggestion
lastSeen: opened chat last time
```